assets/default.game.yaml

```diff
@@ -21,72 +21,50 @@ default_settings:
     # Gamepad controls
     gamepad:
       movement:
-        up:
-          SingleAxis:
-            axis_type:
-              Gamepad: LeftStickY
-            positive_low: 0.1
-            negative_low: -1.0
-        left:
-          SingleAxis:
-            axis_type:
-              Gamepad: LeftStickX
-            positive_low: 1.0
-            negative_low: -0.1
-        down:
-          SingleAxis:
-            axis_type:
-              Gamepad: LeftStickY
-            positive_low: 1.0
-            negative_low: -0.1
-        right:
-          SingleAxis:
-            axis_type:
-              Gamepad: LeftStickX
-            positive_low: 0.1
-            negative_low: -1.0
-      flop_attack:
-        GamepadButton: South
-      shoot:
-        GamepadButton: East
-      throw:
-        GamepadButton: West
+        up: !SingleAxis
+          axis_type: !Gamepad LeftStickY
+          positive_low: 0.1
+          negative_low: -1.0
+
+        left: !SingleAxis
+          axis_type: !Gamepad LeftStickX
+          positive_low: 1.0
+          negative_low: -0.1
+
+        down: !SingleAxis
+          axis_type: !Gamepad LeftStickY
+          positive_low: 1.0
+          negative_low: -0.1
+
+        right: !SingleAxis
+          axis_type: !Gamepad LeftStickX
+          positive_low: 0.1
+          negative_low: -1.0
+      flop_attack: !GamepadButton South
+      shoot: !GamepadButton East
+      throw: !GamepadButton West
```

Comment:
It's also less lines which is nice :shrug:

Reply:
Yeah, overall seems nice.